src/barril/_foundation/odict.py

```diff
@@ -39,12 +39,7 @@ def __delitem__(self, key):
             collections.OrderedDict.__delitem__(self, key)
 
 
-def _GetSymbol():
-    if six.PY2:
-        import _ordereddict
-        return _ordereddict.ordereddict
-    else:
-        return _OrderedDict
-
-
-odict = _GetSymbol()
+if six.PY2:
+    from _ordereddict import ordereddict as odict
+else:
+    odict = _OrderedDict
```

Comment:
I think we could do with `dict` on Python 3.5 onwards (please create a ticket in etk and assign if to me if you don't have the time to check it).

Reply:
Done: https://eden.esss.com.br/jira/browse/ETK-1558